### ISSUE TITLE
Proposal: Formatting options for PureScript code.

### ISSUE THREAD

**Issue by jhodgdev** [mlabs-haskell/styleguide#14]
We need a formatting solution for PureScript code. 

May I suggest [purs-tidy](https://github.com/natefaubion/purescript-tidy) with this config:

```json
{
  "importSort": "ide",
  "importWrap": "auto",
  "indent": 2,
  "operatorsFile": null,
  "ribbon": 1,
  "typeArrowPlacement": "last",
  "unicode": "source",
  "width": 80
}
```

**Comment by kozross** [comment]
@jhodgdev I actually feel a whole separate Purescript document may be in order. Purescript, despite similarities, is _not_ Haskell, and clarity would be nice.

**Comment by jhodgdev** [comment]
Yeah, that sounds like it'd be prudent. If you want any input, I'd be happy to help you on it.
